### PR TITLE
Create com.gitlab.bitseater.meteo

### PR DESCRIPTION
**Pull request by bitseater**

### CODE REVIEW

File: com.gitlab.bitseater.meteo.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "com.gitlab.bitseater.meteo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
```
Comment (bilelmoussaoui) [inline com.gitlab.bitseater.meteo.json]: You should use 3.30 instead.

File: com.gitlab.bitseater.meteo.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.gitlab.bitseater.meteo",
+    "build-options" : {
+        "cflags" : "-O2  -Wno-deprecated-declarations",
```
Comment (bilelmoussaoui) [inline com.gitlab.bitseater.meteo.json]: The -O2 flags are not needed.

File: com.gitlab.bitseater.meteo.json [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                    "sha256" : "4a7e31cf7889669aebf04631543af64435edd989685159b804911c6005db908d"
+                }
+            ]
+        },
+        {
+            "name" : "geocode-glib",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://ftp.gnome.org/pub/GNOME/sources/geocode-glib/3.24/geocode-glib-3.24.0.tar.xz",
```
Comment (bilelmoussaoui) [inline com.gitlab.bitseater.meteo.json]:
```suggestion
                    "url" : "https://ftp.gnome.org/pub/GNOME/sources/geocode-glib/3.24/geocode-glib-3.24.0.tar.xz",
```

File: com.gitlab.bitseater.meteo.json [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        {
+            "name" : "libdbusmenu",
+            "config-opts" : [
+                "--with-gtk=3",
+                "--disable-gtk-doc",
+                "--disable-doc",
+                "--disable-dumper"
+            ],
+            "build-options" : {
+                "env" : {
```
Comment (bilelmoussaoui) [inline com.gitlab.bitseater.meteo.json]: This won't work with GNOME 3.30 runtime, it used to work due to some patches upstream on freedesktop runtime which are not used anymore. You can fix it using make-install-args I guess.

**Comment by nedrichards** [comment]
bot, build com.gitlab.bitseater.meteo

File: com.gitlab.bitseater.meteo.json [hunk 5]
```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "com.gitlab.bitseater.meteo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.gitlab.bitseater.meteo",
+    "build-options" : {
+        "cflags" : "-O2  -Wno-deprecated-declarations",
+        "cxxflags" : "-O2",
+        "env" : {
```
Comment (nedrichards) [inline com.gitlab.bitseater.meteo.json]: ideally clean this up

File: com.gitlab.bitseater.meteo.json (hunk 2 above)
Comment (bitseater) [reply com.gitlab.bitseater.meteo.json]: Hi @bilelmoussaoui . Thanks for your support. -O2 flags only in line 8 . Can i delete line 9 too?
`        "cxxflags" : "-O2",`?

Comment (bitseater) [reply com.gitlab.bitseater.meteo.json]: Ok. i just see the comments by @nedrichards about this.

File: com.gitlab.bitseater.meteo.json (hunk 5 above)
Comment (bitseater) [reply com.gitlab.bitseater.meteo.json]: Cleaned with @bilelmoussaoui  comments. Thanks

File: com.gitlab.bitseater.meteo.json (hunk 4 above)
Comment (bitseater) [reply com.gitlab.bitseater.meteo.json]: Hi @bilelmoussaoui . Are you talking about :
`                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR" : "/app/share/gir-1.0",
                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR" : "/app/lib/girepository-1.0"`
lines?

**Comment by nedrichards** [comment]
bot, build com.gitlab.bitseater.meteo

File: com.gitlab.bitseater.meteo.json (hunk 4 above)
Comment (nedrichards) [reply com.gitlab.bitseater.meteo.json]: Yep, like this: https://github.com/flathub/org.gnome.FeedReader/blob/0ec2f910dd97d892fb7818f978cd33d0c7cb5659/org.gnome.FeedReader.json#L31

**Comment by nedrichards** [comment]
Other than the 3.30 runtime change (and the move to make-install-args that will entail) this lgtm. builds fine on all architectures.

**Comment by bitseater** [comment]
Thanks @nedrichards and @bilelmoussaoui . Now it builds with 3.30 and make-install-args, and all the changes suggested. You can see it at this  [Gitlab pipeline](https://gitlab.com/bitseater/meteo_flatpak/-/jobs/126205997)

File: com.gitlab.bitseater.meteo.json (hunk 1 above)
Comment (bitseater) [reply com.gitlab.bitseater.meteo.json]: 3.30 updated

**Comment by nedrichards** [comment]
bot, build com.gitlab.bitseater.meteo

**Comment by nedrichards** [comment]
Nice!

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.gitlab.bitseater.meteo

**Pull request closed without merging**